posthog/models/filters/mixins/funnel.py

```diff
@@ -1,11 +1,13 @@
 from typing import Optional
 
-from posthog.constants import FUNNEL_FROM_STEP, FUNNEL_STEP, FUNNEL_TO_STEP, FUNNEL_WINDOW_DAYS
+from django.db.models import base
+
+from posthog.constants import FUNNEL_FROM_STEP, FUNNEL_STEP, FUNNEL_TO_STEP, FUNNEL_TYPE, FUNNEL_WINDOW_DAYS, FunnelType
 from posthog.models.filters.mixins.base import BaseParamMixin
 from posthog.models.filters.mixins.utils import cached_property, include_dict
 
 
-class FunnelFromToStepsMixin(BaseParamMixin):
+class FunnelTrendsFromToStepsMixin(BaseParamMixin):
```

Comment:
renamed these, because it was getting confusing otherwise: I couldn't know what these are for just by the name.

Reply:
Hm, actually I reuse this in https://github.com/PostHog/posthog/pull/4947 (see code), so I'd prefer keeping the slightly more ambiguous name

Reply:
Will only change FunnelSteps then! This makes sense :)